### PR TITLE
[bn] Localization of service-account.md

### PR DESCRIPTION
**Pull request by Ashfinn**
Resolves : #45729

### CODE REVIEW

**Review by sajibAdhi — CHANGES_REQUESTED** [review]

File: content/bn/docs/reference/glossary/service-account.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: সার্ভিস অ্যাকাউন্ট
+id: service-account
+date: 2018-04-12
+full_link: /docs/tasks/configure-pod-container/configure-service-account/
+short_description: >
+  পডগুলিতে চলমান কার্যক্রমের জন্য একটি পরিচিতি সরবরাহ করে।
```
[the remark this answers is not in the document]
Comment (Ashfinn) [reply content/bn/docs/reference/glossary/service-account.md]: fixed it

File: content/bn/docs/reference/glossary/service-account.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+tags:
+- fundamental
+- core-object
+---
+
+{{< glossary_tooltip text="পডগুলিতে" term_id="pod" >}} চলমান কার্যক্রমের জন্য একটি পরিচিতি সরবরাহ করে।
+
+<!--more--> 
+
+যখন পডগুলির ভেতরের প্রক্রিয়াগুলি ক্লাস্টার অ্যাক্সেস করে, তাদেরকে প্রাসঙ্গিক সার্ভিস অ্যাকাউন্ট হিসেবে API সার্ভার দ্বারা অথেনটিকেট(authenticate) করা হয়, উদাহরণস্বরূপ, `default`। যদি আপনি সার্ভিস অ্যাকাউন্ট নির্দিষ্ট না করে একটি পড তৈরি করেন, তাহলে স্বয়ংক্রিয়ভাবে একই {{< glossary_tooltip text="নেমস্পেস" term_id="namespace" >}} এ ডিফল্ট সার্ভিস অ্যাকাউন্টটি নির্ধারন করে দেওয়া হয়।
```
[the remark this answers is not in the document]
Comment (Ashfinn) [reply content/bn/docs/reference/glossary/service-account.md]: fixed it too

**Review by mitul3737 — APPROVED** [review]

**Review by sajibAdhi — APPROVED** [review]
/lgtm

**Review by Imtiaz1234 — APPROVED** [review]

**Pull request merged**